examples/BuildOutOfContainer/Startup.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.ApplicationInsights.Kubernetes.Debugging;
+using Microsoft.OpenApi.Models;
+using System.Diagnostics;
+
+namespace BuildOutOfContainer
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Application Insights connection string is set in appsettings.json
+            services.AddApplicationInsightsTelemetry();
+            // Skip the following lines when it is in Production environment
```

Comment:
Is there an equivalent of `env.IsDevelopment()` that you could use here?

Reply:
The way I can think of is injecting `IWebHostEnvironment`. Will update.

Reply:
Fixed.